### ISSUE TITLE
Storage Explorer connect error "ERR_SSL_KEY_USAGE_INCOMPATIBLE"

### ISSUE THREAD

**Issue by MarioHelbingDB** [microsoft/AzureStorageExplorer#7705]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1

### Architecture

x64

### Platform

Windows

### Bug Description

I can't connect to my storage account with the following error:
ERR_SSL_KEY_USAGE_INCOMPATIBLE
I don't change my storage account and from one to a other day I can't connect.

### Steps to Reproduce

1.Lunch Explorer
2. open blob containers
3. I have the error

### Actual Experience

I can't connect to my storage account.

**Comment by MarioHelbingDB** [comment]
We use Azure Stack Edge and I want connect to the storage account of the Azure Stack Edge.

**Comment by MarioHelbingDB** [comment]
we had a connection and we have worked, the complete issue message:
ProducerError:{
  "name": "Electron Net Error",
  "message": "net::ERR_SSL_KEY_USAGE_INCOMPATIBLE",
  "stack": "Error: net::ERR_SSL_KEY_USAGE_INCOMPATIBLE\n    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:2:50234)\n    at SimpleURLLoaderWrapper.emit (node:events:513:28)",
  "cause": {}
}

**Comment by MRayermannMSFT** [comment]
@MarioHelbingDB can you share what steps you have already followed in attempting to connect to your azure stack edge?

**Comment by MRayermannMSFT** [comment]
Closing due to lack of response. This issue will no longer be monitored. If you need further help, we recommend opening a support ticket via the Azure portal. Alternatively you can open a new issue here.

**Comment by MarioHelbingDB** [comment]
1)	We checked the ASE certificates were not expired, we (re)installed the same on the Windows VM (including the ASE Blob certificate), and we confirmed that all entries were correct in the Windows hosts file (including each specific ASE blob storage).
2)	Then, in Storage Explorer (version 1.32.1), we selected Edit -> Target Azure Stack APIs -> Restart the Storage Explorer.
3)	We tried without explicitly importing the certificates in Storage Explorer, but also tried after importing the same there (in Edit -> SSL Certificates -> Import).
4)	We Attached a resource -> Storage Account -> Account name and key -> We filled-in some display name, set the correct Account name (already created in ASE and having a container), and set the account key. The Storage Domain was set to <devicename>.<domain>.
5)   If we set the checkmark "use http" while connecting to the storage account we can establish the connection but if we use https then the error occurs.
